SDPi_Supplement/asciidoc/volume2/dev-28/tf2-dev-28.adoc

```diff
@@ -0,0 +1,65 @@
+:var_transaction_id: DEV-28
+
+:var_label_dev_28_message_contextreport: Notification
+
+
+[#vol2_clause_dev_28]
+=== Notify Change in System Context and Capabilities [{var_transaction_id}]
+
+==== Scope
+
+include::tf2-dev-28-summary.adoc[]
+
+
+==== Actor Roles
+
+.Actor Roles {var_transaction_id}
+[cols="1,2"]
+|===
+|Actor |Roles
+
+|<<actor_somds_consumer>>
+|Listens for a <<vol2_clause_dev_28_message_notification, {var_label_dev_28_message_contextreport}>> message to retrieve context updates.
+
+|<<actor_somds_provider>>
+|While a subscription is running, the <<actor_somds_provider>> delivers <<vol2_clause_dev_28_message_notification, {var_label_dev_28_message_contextreport}>> messages.
+
+|===
+
+==== Referenced Standards
+
+* <<ieee_11073_10207_2017>> Section 7.4 Message Model
+
+==== Messages
+
+.Message Interaction Diagram {var_transaction_id}
+[plantuml#vol2_figure_dev_28_sequence, target=puml-dev-28-sequence, format=svg, reftext='{figure-caption} {counter:refnum}',width=100%]
+....
+include::../../plantuml/vol2-figure-dev-28-sequence.puml[]
+....
+
+
+// ---------- EpisodicContextReport ---------
+
+
+[#vol2_clause_dev_28_message_notification]
+===== {var_label_dev_28_message_contextreport} Message
+
+The {var_label_dev_28_message_contextreport} message contains updated context data and is delivered by a <<actor_somds_provider>> to subscribed <<actor_somds_consumer>>s.
+
+[#vol2_clause_dev_28_message_notification_trigger_events]
+====== Trigger Events
+
+The {var_label_dev_28_message_contextreport} message is sent whenever a <<actor_somds_consumer>> is subscribed to a <<actor_somds_provider>> and intends to get notified on certain changes of a <<actor_somds_provider>>.

```

Comment:
Something along the lines:
```suggestion
The {var_label_dev_28_message_contextreport} message is sent whenever a context of a <<actor_somds_provider>> is updated and a <<actor_somds_consumer>> is subscribed to context reports of a <<actor_somds_provider>>.
```
To make clear that a trigger is also bound to the update of the context.